front/components/items/Appointment.tsx

```diff
@@ -48,6 +65,24 @@ const Appointment = () => {
         return allDates?.indexOf(item) === index;
     });
     const availableDate = stringDate?.map((item) => new Date(item));
+    const randomDate = new Date(1999, 1, 1);
+    const availableDatewoPastDates = availableDate?.map((item) => {
```

Comment:
datewo? I believe this is typo. Also its hard to understand what the variable contains after reading the name